### ISSUE TITLE
Cant build android, > com.android.ide.common.process.ProcessException: Failed to execute aapt

### ISSUE THREAD

**Issue by tayfunyasar** [infinitered/ignite#1400]
---
I'm having an error when I try to "build" android.

```
Tayfuns-MacBook-Pro:app-mobile tayfunyasar$ cd android/
Tayfuns-MacBook-Pro:android tayfunyasar$ ./gradlew clean build

> Configure project :app 
WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html

> Configure project :react-native-config 
WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html
WARNING: The specified Android SDK Build Tools version (23.0.1) is ignored, as it is below the minimum supported version (27.0.3) for Android Gradle Plugin 3.1.4.
Android SDK Build Tools 27.0.3 will be used.
To suppress this warning, remove "buildToolsVersion '23.0.1'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.

> Configure project :react-native-device-info 
WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html
WARNING: The specified Android SDK Build Tools version (25.0.2) is ignored, as it is below the minimum supported version (27.0.3) for Android Gradle Plugin 3.1.4.
Android SDK Build Tools 27.0.3 will be used.
To suppress this warning, remove "buildToolsVersion '25.0.2'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.

> Configure project :react-native-gesture-handler 
WARNING: Configuration 'provided' is obsolete and has been replaced with 'compileOnly'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html
WARNING: The specified Android SDK Build Tools version (25.0.2) is ignored, as it is below the minimum supported version (27.0.3) for Android Gradle Plugin 3.1.4.
Android SDK Build Tools 27.0.3 will be used.
To suppress this warning, remove "buildToolsVersion '25.0.2'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.

> Task :react-native-i18n:compileDebugJavaWithJavac 
Note: /Users/tayfunyasar/dev/gitlap/app/client/app-mobile/node_modules/react-native-i18n/android/src/main/java/com/AlexanderZaytsev/RNI18n/RNI18nModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

> Task :app:bundleReleaseJsAndAssets 
warning: the transform cache was reset.
Loading dependency graph, done.
bundle: Writing bundle output to: /Users/tayfunyasar/dev/gitlap/app/client/app-mobile/android/app/build/generated/assets/react/release/index.android.bundle
bundle: Done writing bundle output
bundle: Copying 54 asset files
bundle: Done copying assets

> Task :react-native-i18n:compileReleaseJavaWithJavac 
Note: /Users/tayfunyasar/dev/gitlap/app/client/app-mobile/node_modules/react-native-i18n/android/src/main/java/com/AlexanderZaytsev/RNI18n/RNI18nModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

error: resource android:style/TextAppearance.Material.Widget.Button.Borderless.Colored not found.
error: resource android:style/TextAppearance.Material.Widget.Button.Colored not found.
/Users/tayfunyasar/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/baf05ba217c6ee9c35f9fb008fc2ffca/res/values-v26/values-v26.xml:9:5-12:13: AAPT: error: resource android:attr/colorError not found.
    
/Users/tayfunyasar/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/baf05ba217c6ee9c35f9fb008fc2ffca/res/values-v26/values-v26.xml:13:5-16:13: AAPT: error: resource android:attr/colorError not found.
    
/Users/tayfunyasar/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/baf05ba217c6ee9c35f9fb008fc2ffca/res/values-v26/values-v26.xml:17:5-93: AAPT: error: style attribute 'android:attr/keyboardNavigationCluster' not found.
    
/Users/tayfunyasar/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/baf05ba217c6ee9c35f9fb008fc2ffca/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/fontStyle not found.
    
/Users/tayfunyasar/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/baf05ba217c6ee9c35f9fb008fc2ffca/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/font not found.
    
/Users/tayfunyasar/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/baf05ba217c6ee9c35f9fb008fc2ffca/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/fontWeight not found.
    
error: failed linking references.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-config:verifyReleaseResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 2s
172 actionable tasks: 163 executed, 9 up-to-date
Tayfuns-MacBook-Pro:android tayfunyasar$ 
```

---
Steps to reproduce
rm -rf node_modules && npm I
cd android
./gradlew clean build

---
`ignite doctor` results:
```
Tayfuns-MacBook-Pro:android tayfunyasar$ ignite doctor
System
  platform           darwin                                                                                                                      
  arch               x64                                                                                                                         
  cpu                8 cores                                                                 Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz  
  directory          /Users/tayfunyasar/dev/gitlap/app/client/app-mobile/android                                                       

JavaScript
  node               11.12.0      /usr/local/bin/node  
  npm                6.7.0        /usr/local/bin/npm   
  yarn               1.13.0       /usr/local/bin/yarn  

React Native
  react-native-cli   2.0.1       

Ignite
  ignite             2.2.3        /usr/local/bin/ignite  

Android
  java               1.8.0_202    /Users/tayfunyasar/.jenv/shims/java     
  android home       -            /Users/tayfunyasar/Library/Android/sdk  

iOS
  xcode              10.1       
```

**Comment by jamonholmgren** [comment]
@tayfunyasar Is this an Andross or Bowser app?

**Comment by tayfunyasar** [comment]
Andross app

**Comment by ryanlntn** [comment]
This still appears to be an issue. To clarify `react-native run-android` works fine but when trying to run `./gradlew clean build` directly it will error. This appears to be due to an outdated version of `react-native-device-info`. Upgrading to the latest resolves the issue.

**Comment by ryanlntn** [comment]
This is now resolved with the latest ignite-andross (4.0.1)
